tests/testthat/shinyapp-with-absolute-paths/server.R

```diff
@@ -28,8 +28,8 @@ shinyServer(function(input, output) {
     hist(x, breaks = bins, col = "darkgray", border = "white")
 
     ## read a csv file
-    file <- read.csv("college.txt") ## bad
-    file <- read.csv("College.txt") ## okay
+    file <- read.csv("data/college.txt") ## bad
```

Comment:
Maybe also add a test demonstrating that we do not detect problems with a current-directory `college.txt`? Very weak nit.